Integrations/integration-TruSTAR.yml

```diff
@@ -33,6 +33,26 @@ configuration:
   defaultvalue: "false"
   type: 8
   required: false
+- display: File Threshold (LOW, MEDIUM, HIGH)
+  name: file_threshold
+  defaultvalue: MEDIUM
+  type: 0
+  required: false
+- display: URL Threshold (LOW, MEDIUM, HIGH)
+  name: url_threshold
+  defaultvalue: MEDIUM
+  type: 0
+  required: false
+- display: IP Threshold (LOW, MEDIUM, HIGH)
+  name: ip_threshold
+  defaultvalue: MEDIUM
+  type: 0
+  required: false
+- display: Domain Threshold (LOW, MEDIUM, HIGH)
```

Comment:
Should be as follows: `Domain Threshold. Minimum TruSTAR priority level to consider the domain malicious.`